src/kvstore/SnapshotManagerImpl.cpp

```diff
@@ -17,7 +17,7 @@ void SnapshotManagerImpl::accessAllRowsInSnapshot(GraphSpaceID spaceId,
                                                   raftex::SnapshotCallback cb) {
     CHECK_NOTNULL(store_);
     std::unique_ptr<KVIterator> iter;
-    auto prefix = NebulaKeyUtils::prefix(partId);
+    auto prefix = NebulaKeyUtils::snapshotPrefix(partId);
```

Comment:
If the prefix is empty,  what's the behavior for seek of rocksdb.

Reply:
Just the same as SeekToFirst and iterate over all.